### PR TITLE
l10n: Add Italian translation

### PR DESCRIPTION
**Pull request by MiloCasagrande**
Signed-off-by: Milo Casagrande <milo@milo.name>

### CODE REVIEW

**Review by tchx84 — CHANGES_REQUESTED** [review]
This is great :+1: 

A small comment though, can you remove the translations marked with `#. TRANSLATORS: Don't translate this text`.

Thanks!

**Comment by MiloCasagrande** [comment]
> A small comment though, can you remove the translations marked with `#. TRANSLATORS: Don't translate this text`.

I can, but why would I need to do that? 

I copied the original string in there and it's kind of normal (at least as GNOME translations go) to do just that, especially if you put a comment before stating to "not translate".

If a string must not be translated at all it should not be made available in the translation template file (the POT file). Or am I missing something?

**Comment by tchx84** [comment]
> > A small comment though, can you remove the translations marked with `#. TRANSLATORS: Don't translate this text`.
> 
> I can, but why would I need to do that?
> 
> I copied the original string in there and it's kind of normal (at least as GNOME translations go) to do just that, especially if you put a comment before stating to "not translate".
> 
> If a string must not be translated at all it should not be made available in the translation template file (the POT file). Or am I missing something?

I see it the other way around :)

For instance, we don't have a completion % metric that we care about, there is no technical reason for copying the same string twice, and it open the possibility of introducing typos to sensitive strings like the app-id.

I need to double check whether is possible to exclude these automatically, but until then I rather have easier reviews hehe...

**Comment by tchx84** [comment]
> I need to double check whether is possible to exclude these automatically

@MiloCasagrande maybe you can help me with that?

**Comment by MiloCasagrande** [comment]
> @MiloCasagrande maybe you can help me with that?

On the desktop entry spec the `Icon` key is a `iconstring` value type, and I assumed it was not going to be translatable. Apparently it's not.

On the other hand, the `Name` key is a `localestring` and it will be automatically set as translatable.

The old way with `intltool` was to mark the key with an underscore to make it translatable (e.g. `_Name=`), but everybody is moving to (x)gettext now and it just takes everything that might be translatable by default.

My google-fu didn't turn up anything more useful on the subject, unfortunately.

**Comment by MiloCasagrande** [comment]
> For instance, we don't have a completion % metric that we care about, there is no technical reason for copying the same string twice, and it open the possibility of introducing typos to sensitive strings like the app-id.

True, although it's easy to spot them in a PR.

Anyway, I pushed another commit to fix it in the Italian translation.

File: po/it.po [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+"Flatseal is a graphical utility to review and modify basic permissions from "
+"your Flatpak applications."
+msgstr ""
+"Flatseal è uno strumento grafico per visualizzare e modificare i permessi di "
+"base delle applicazioni Flatpak."
+
+#. TRANSLATORS: Don't translate this text
+#: data/com.github.tchx84.Flatseal.appdata.xml.in:31
+msgid "Martin Abente Lahaye"
+msgstr "Martin Abente Lahaye"
```
Comment (tchx84) [inline po/it.po]: :point_up: :pray:

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: po/it.po (hunk 1 above)
Comment (MiloCasagrande) [reply po/it.po]: Missed that one.
It's now fixed.

File: po/it.po [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+msgid "Access PulseAudio sound server"
+msgstr "Accesso al server audio PulseAudio"
+
+#: src/model.js:65
+msgid "Access D-Bus system bus (unrestricted)"
+msgstr "Accesso al bus di sistema D-Bus (senza restrizioni)"
+
+#: src/model.js:70
+msgid "Access D-Bus session bus (unrestricted)"
+msgstr "Accesso al bus di sistema D-Bus (senza restrizioni)"
```
Comment (tchx84) [inline po/it.po]: Catched this last minute, while testing it, shouldn't it be ` sessione` ?

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: po/it.po (hunk 2 above)
Comment (MiloCasagrande) [reply po/it.po]: And fixed. Thanks for catching that one.

**Comment by tchx84** [comment]
Thanks, and for the research as well.

**Pull request merged**